server/controllers/project.controller/createProject.js

```diff
@@ -30,3 +31,27 @@ export default function createProject(req, res) {
     .then(populateUserData)
     .catch(sendFailure);
 }
+
+// TODO: What happens if you don't supply any files?
```

Comment:
i think there are two options here: (1) create default files (the ones that are currently the default when you click sketch-> new) or (2) the project isn't valid. my instinct here is (1)...

Reply:
This is done. See ee6891d.